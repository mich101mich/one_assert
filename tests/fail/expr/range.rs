fn main() {
    one_assert::assert!(false..);
    one_assert::assert!(..=5);
}
