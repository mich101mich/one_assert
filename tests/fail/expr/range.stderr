error[E0308]: mismatched types
 --> tests/fail/expr/range.rs:2:25
  |
2 |     one_assert::assert!(false..);
  |                         ^^^^^^^ expected `bool`, found `RangeFrom<bool>`
  |
  = note: expected type `bool`
           found struct `RangeFrom<bool>`

error[E0308]: mismatched types
 --> tests/fail/expr/range.rs:3:25
  |
3 |     one_assert::assert!(..=5);
  |                         ^^^^ expected `bool`, found `RangeToInclusive<{integer}>`
  |
  = note: expected type `bool`
           found struct `RangeToInclusive<{integer}>`
